### PR TITLE
[A]Fix potential crash when calling ClearFocus in SearchBarRenderer

### PR DESCRIPTION
**Pull request by jassmith**
### Description of Change ###

Adds a try/catch around API that fails in previewer specific situation.

### Bugs Fixed ###

None

### API Changes ###

None

### Behavioral Changes ###

Works now with previewer

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of master at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Platform.Android/Renderers/SearchBarRenderer.cs [hunk 1]
```diff
@@ -148,14 +148,26 @@ void UpdateEnabled()
...
 
+		void ClearFocus(SearchView view)
+		{
+			try
+			{
+				view.ClearFocus();
+			}
+			catch (Java.Lang.UnsupportedOperationException)
+			{
+				// silently catch these as they happen in the previewer due to some bugs in upstread android
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Platform.Android/Renderers/SearchBarRenderer.cs]: upstream

**Comment by alanmcgovern** [comment]
Can we get this backported to 2.3.4?

**Comment by rmarinho** [comment]
done @alanmcgovern should go on 2.3.4-pre6

**Comment by alanmcgovern** [comment]
Gracias!

**Pull request merged**